.github/workflows/build.yml

```diff
@@ -0,0 +1,112 @@
+name: Build
+
+on: push
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, windows-latest ]
+        pg_major: [ 11, 10 ]
+        include:
+          - pg_major: 11
+            edb_version: '11.4-2'
+            postgis_version: '2.5.3'
+          - pg_major: 10
+            edb_version: '10.9-2'
+            postgis_version: '2.5.3'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Start PostgreSQL ${{ matrix.pg_major }} (Linux)
+        if: startsWith(matrix.os, 'ubuntu')
+        run: docker run --detach --name postgresql -p 5432:5432 npgsql/postgres:${{ matrix.pg_major }}
```

Comment:
One more thing... I know we had a conversation about this before, but I still don't see the value of using a docker image here rather than setting things up directly. I think it's better and simpler if we just have all the setup knowledge here in this script, rather than having to set up new docker images; PostgreSQL also has superb apt sources so this should be an extremely easy thing to do.

I don't think we should block on this - am totally fine with merging the PR as-is and discussing later.